vector/src/main/java/im/vector/app/features/matrixto/MatrixToRoomSpaceFragment.kt

```diff
@@ -81,6 +81,15 @@ class MatrixToRoomSpaceFragment @Inject constructor(
                         if (peek.roomType == RoomType.SPACE) {
                             views.matrixToBetaTag.isVisible = true
                             avatarRenderer.renderSpace(matrixItem, views.matrixToCardAvatar)
+                            if (peek.isPublic) {
+                                views.matrixToAccessText.setTextOrHide(context?.getString(R.string.public_space))
+                                views.matrixToAccessImage.isVisible = true
+                                views.matrixToAccessImage.setImageResource(R.drawable.ic_public_room)
+                            } else {
+                                views.matrixToAccessText.setTextOrHide(context?.getString(R.string.private_space))
+                                views.matrixToAccessImage.isVisible = true
```

Comment:
`views.matrixToAccessImage.isVisible = true` is the same on `if` and `else` block. Could be simplified, maybe in the layout